recipes/mo-catnip/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mo-catnip" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 491e72214f2018dcd57e580956c97eb91b6df1cd6172b85a7e9e0da985733e54
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - matplotlib ==2.2.3
+    - numpy ==1.15.4
+    - pip
+    - python
+    - scipy ==1.2.1
+    - six ==1.12.0
+  run:
+    - matplotlib ==2.2.3
+    - numpy ====1.15.4
+    - python
+    - scipy ==1.2.1
+    - six ==1.12.0
+
+test:
+  imports:
+    - catnip
+
+about:
+  home: "https://github.com/MetOffice/CATNIP"
+  license: BSD
```

Comment:
Use the appropriate spdx abbreviation:

https://spdx.org/licenses/

Reply:
change the license name with appropriate spdx abbreviation